packages/epics/src/agreements/components/agreements-list.tsx

```diff
@@ -2,29 +2,45 @@ import { FC } from 'react';
 import { AgreementCard } from './agreement-card';
 import { useAgreements } from '../hooks/use-agreements';
 import Link from 'next/link';
+import { AgreementItem } from '@hypha-platform/graphql/rsc';
 
 type AgreementsListProps = {
   page: number;
   activeFilter: string;
   basePath: string;
+  agreementsProp?: AgreementItem[];
```

Comment:
Let's not replicating the Props here. Just `agreements`

Reply:
I added this to name because agreements is already used (it's the variable name for the list of agreements that come from the hook) and agreementsProp is the external passed in. I made it so that I could reuse this component in two different places

Reply:
Ok, that is even more confusing. We need to resolve that. Why do we need both?